src/java/com/puppetlabs/ssl_utils/ExtensionsUtils.java

```diff
@@ -616,8 +619,10 @@ private static Object asn1ObjToObj(ASN1Encodable asn1Prim)
             X500Name name = (X500Name) asn1Prim;
             return name.toString();
         } else {
-            // Return the raw data if there's no clear method of decoding
-            return asn1Prim.toASN1Primitive().getEncoded();
```

Comment:
Hmm...yeah if it's not required for the ticket, not sure we have enough reason to believe this form is any better than the raw data?

I think it'd be better to just leave it as it was